package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "eslint-config-extensionengine",
+  "version": "1.0.0",
+  "description": "Extension Engine's eslint config",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ExtensionEngine/eslint-config-extensionengine.git"
+  },
+  "scripts": {
+    "lint": "eslint ."
+  },
+  "keywords": [
+    "eslint",
+    "eslintconfig",
+    "config",
+    "Extension Engine",
+    "javascript",
+    "vue",
+    "node"
+  ],
+  "author": "Zdravko Ćurić (https://github.com/zcuric)",
```

Comment:
```suggestion
  "author": "ExtensionEngine <info@extensionengine.com>",
```

Reply:
![Alt Text](https://media.giphy.com/media/xT1XH0lC7yoLLB1BSM/giphy.gif)

Reply:
Because we obviously have a website we should put it there 😉 https://docs.npmjs.com/files/package.json#people-fields-author-contributors